src/aviary/core.py

```diff
@@ -15,6 +15,7 @@
     TaskEnvClientState,
     TaskEnvironmentClient,
 )
+from aviary.functional import fenv
```

Comment:
Can you add `DynamicState` here, it will be a common import for those adding type hints